src/features/pass-survey/model/hooks/useActivityStepper.ts

```diff
@@ -11,8 +11,14 @@ function useActivityStepper(state: ActivityState | undefined) {
   const isTutorialStep = currentPipelineItem?.type === 'Tutorial';
   const isLastStep = items && step === items.length - 1;
 
+  const additionalAnswerRequired =
+    currentPipelineItem?.additionalText?.required;
+
   const canMoveNext =
-    isTutorialStep || currentPipelineItem?.isSkippable || !!answers[step];
+    isTutorialStep ||
+    currentPipelineItem?.isSkippable ||
+    (!!answers[step]?.answer &&
```

Comment:
Move to variables:
const hasAnswer =  !!answers[step]?.answer
const hasAdditionalAnswer =  !!answers[step]?.additionalAnswer